### PR TITLE
LZMA Decompression

### PR DESCRIPTION
**Pull request by alexanderturner**
- Added LZMA decompression for control files
- Updated readme 
- Added lzma test for control file 
- removed control file test on `TestInspectPackage` in favour of control file tests given the two different control files, and error should suffice here

### CODE REVIEW

**Comment by alexanderturner** [comment]
Hey there :) Any opportunity to look into this one?

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM alpine:latest
+
+# Set the Current Working Directory inside the container
+WORKDIR $GOPATH/src/github.com/esell/deb-simple
+
+# Copy everything from the current directory to the PWD (Present Working Directory) inside the container
+COPY deb-simple /
+
+# This container exposes port 8080 to the outside world
```
Comment (esell) [inline Dockerfile:9]: One very minor nitpick @alexanderturner , any chance you can update this to match the port being exposed for clarity sake?

Comment (alexanderturner) [reply Dockerfile:9]: Done :) Sorry about that!

**Comment by esell** [comment]
Thank you for this!

**Pull request merged**